spec.html

```diff
@@ -18,6 +18,65 @@ <h1>Array Objects</h1>
         <emu-clause id="sec-properties-of-the-array-prototype-object">
             <h1>Properties of the Array Prototype Object</h1>
 
+            <emu-clause id="sec-array.prototype.sort">
+                <h1>Array.prototype.sort ( _comparefn_ )</h1>
+
+                <emu-note type="editor">This algorithm has not been updated. The only difference is that _comparefn_ is now passed explicitly to SortCompare.</emu-note>
+
+                <p>When the `sort` method is called, the following steps are taken:</p>
+                <emu-alg>
+                    1. [id="step-array-sort-comparefn"] If _comparefn_ is not *undefined* and IsCallable(_comparefn_) is *false*, throw a *TypeError* exception.
+                    1. Let _obj_ be ? ToObject(*this* value).
+                    1. [id="step-array-sort-len"] Let _len_ be ? LengthOfArrayLike(_obj_).
+                    1. Let _items_ be a new empty List.
+                    1. Let _k_ be 0.
+                    1. Repeat, while _k_ &lt; _len_,
+                        1. Let _Pk_ be ! ToString(𝔽(_k_)).
+                        1. Let _kPresent_ be ? HasProperty(_obj_, _Pk_).
+                        1. If _kPresent_ is *true*, then
+                        1. Let _kValue_ be ? Get(_obj_, _Pk_).
+                        1. Append _kValue_ to _items_.
+                        1. Set _k_ to _k_ + 1.
+                    1. Let _itemCount_ be the number of elements in _items_.
+                    1. [id="step-array-sort"] Sort _items_ using an implementation-defined sequence of calls to SortCompare <ins>with arguments &laquo; _comparefn_, _a_, _b_ &raquo;, where _a_ and _b_ are the two elements of _items_ to compare</ins>. If any such call returns an abrupt completion, stop before performing any further calls to SortCompare or steps in this algorithm and return that completion.
+                    1. Let _j_ be 0.
+                    1. Repeat, while _j_ &lt; _itemCount_,
+                        1. Perform ? Set(_obj_, ! ToString(𝔽(_j_)), _items_[_j_], *true*).
+                        1. Set _j_ to _j_ + 1.
+                    1. Repeat, while _j_ &lt; _len_,
+                        1. Perform ? DeletePropertyOrThrow(_obj_, ! ToString(𝔽(_j_))).
+                        1. Set _j_ to _j_ + 1.
+                    1. Return _obj_.
+                </emu-alg>
+
+                <emu-clause id="sec-sortcompare" type="abstract operation">
+                    <h1>
+                        SortCompare (
+                            <ins>_comparefn_</ins>: <ins>an ECMAScript language value</ins>,
+                            _x_: unknown,
+                            _y_: unknown,
+                        )

```

Comment:
For now this is rendered like this:
![Schermata da 2022-01-03 18-21-55](https://user-images.githubusercontent.com/7000710/147960305-9a39732c-5ef9-4a62-999a-9b076001f426.png)

It will look better when upgrading ecmarkup (https://github.com/tc39/proposal-change-array-by-copy/issues/46)